src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -365,6 +364,41 @@ public async Task UseCallback_BadCertificate_ExpectedPolicyErrors(string url, Ss
             }
         }
 
+        [Fact]
+        public async Task UseCallback_SelfSignedCertificate_ExpectedPolicyErrors()
```

Comment:
This looks awfully similar to `UseCallback_BadCertificate_ExpectedPolicyErrors_Helper`. However, it might lead to less readable state, so probably not worth the effort.

Reply:
I was originally thinking about updating all three variants but realized that would be hard since badssl.com CA is already trusted. And since this is all local, I hope I don't needle the Winhttp things. I plan to look at more ssl tests for 6.